examples/kotlinExample/build.gradle

```diff
@@ -1,7 +1,8 @@
 buildscript {
-    ext.kotlin_version = '1.0.0-rc-1036'
+    ext.kotlin_version = '1.0.0'
     repositories {
         jcenter()
+        mavenCentral()
```

Comment:
The kotlin plugin for kotlin adds mavenCentral. If we wnat to remove it, we can try, but it's not what the plugin does. Thoughts?